README.md

```diff
@@ -1,21 +1,17 @@
 # D://
 _Bitcoin dynamic name protocol_
 
-
-
-
 ## Benefit
 A D:// transaction refers to another transaction/file and thus forms a layer which state can be overwritten. This way, referencing content via D:// links (instead of b:// or c://) has the advantage of keeping the URL while the content can change.
 
-Example B:// (**can not be changed** once set):<br>
-``<img src="B://<TxID(1)>"``
+Example B:// (**can not** be changed once set):<br>``<img src="B://<TxID>"``
 
-Example D:// (Pointer **can be changed**, because the state of the entry can be overwritten.):<br>  
-``<img src="D://<OwnerBitcoinAddress>/<key>"``
+Example D:// (Pointer **can** be changed, because the state can be updated):<br>``<img src="D://<OwnerBitcoinAddress>/<key>"``
```

Comment:
What about this as another example:

`D://<OwnerBitcoinAddress>`

Reply:
good point.